RSSReader-Maruyama/Modules/Login/LoginViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  LoginViewController.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+import AuthenticationServices
+import FBSDKCoreKit
+import FBSDKLoginKit
+import GoogleSignIn
+
+class LoginViewController: UIViewController {
+    
+    @IBOutlet weak var loginButtonStackView: UIStackView!
+    private let SELECT_FEEDS_SEGUE_ID = "selectFeeds"
+    
+    fileprivate let userDataManager = UserDataManager()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupLoginButtons()
+    }
+    
+//MARK:- Setup SignIn Button
+    private func setupLoginButtons() {
+        // Facebook
+        let button_fb = LoginButton()
+        button_fb.custom(text: "Facebookアカウントでログイン", icon: "fb")
+        button_fb.addTarget(self, action: #selector(didPushLoginButtonFacebook), for: .touchUpInside)
+        
+        // Google
+        let button_google = LoginButton()
+        button_google.custom(text: "Googleアカウントでログイン", icon: "google")
+        button_google.addTarget(self, action: #selector(didPushLoginButtonGoogle), for: .touchUpInside)
+        GIDSignIn.sharedInstance()?.presentingViewController = self
+
+        // Apple
+        let button_apple = ASAuthorizationAppleIDButton()
+        button_apple.addTarget(self, action: #selector(didPushLoginButtonApple), for: .touchUpInside)
+        button_apple.heightAnchor.constraint(equalToConstant: 45).isActive = true
+        
+        loginButtonStackView.addArrangedSubview(button_fb)
+        loginButtonStackView.addArrangedSubview(button_google)
+        loginButtonStackView.addArrangedSubview(button_apple)
+    }
+    
+//    MARK: Facebook SignIn Action
```

Comment:
```suggestion
    // MARK: - Facebook SignIn Action

```
Markコメントの書き方は統一してください